galaxy_ng/app/api/ui/viewsets/synclist.py

```diff
@@ -12,4 +12,4 @@
 class SyncListViewSet(api_base.ModelViewSet):
     queryset = models.SyncList.objects.all()
     serializer_class = serializers.SyncListSerializer
-    permission_classes = [access_policy.SyncListAccessPolicy]
+    permission_classes = [access_policy.MySyncListAccessPolicy]
```

Comment:
Since this viewset links to `MySyncListAccessPolicy` now, I'm not sure anything in the code references `SyncListAccessPolicy` so not sure it's needed but it is not really hurting much.